src/Mod/Draft/draftguitools/gui_snapper.py

```diff
@@ -1129,16 +1142,27 @@ def setCursor(self, mode=None):
         else:
             if mode != self.cursorMode:
                 baseicon = QtGui.QPixmap(":/icons/Draft_Cursor.svg")
-                newicon = QtGui.QPixmap(32, 24)
+                devicePixelRatio = self.devicePixelRatio()
+                fullIconSize = self.getCursorSize()
+                newIconWidth = fullIconSize * devicePixelRatio
+                newIconHeight = 0.75 * fullIconSize * devicePixelRatio
```

Comment:
For new variables added, please use `snake_case` as per the Draft coding conventions. Ideally we should change this style in all our modules, but it takes time.

Reply:
Does PySide use snake case?

Reply:
Being consistent is rather difficult
https://stackoverflow.com/questions/3647518/pep8-and-pyqt-how-to-reconcile-function-capitalization

Reply:
Yep, PySide is inconsistent, but we in FreeCAD should strive to use `snake_case` like a proper Python software. Moreover, it's been noted by Qt that the next version, Qt6, will have a more Pythonic interface. I suspect that they will work on this as well.

Reply:
@vocx-fc fixed

Reply:
Thanks. I know it's a pain in the ass, but somebody has to try to keep Python happy.